scripts/linters/pylint_extensions.py

```diff
@@ -543,6 +543,11 @@ class DocstringParameterChecker(checkers.BaseChecker):
             ' form: *args: list(*). Description.',
             'malformed-args-argument',
             'The parameter is incorrectly formatted.'
+        ),
+        'W9039': (
+            'Docstring should be formatted in the following form: Argument:',
```

Comment:
Can you please provide a better description of what this warning is about? Something in which the msg indicates starting from the next line (something around this which makes it clearer).
Also update the `displayed-message` (1st) and `message-help` (3rd) to capture the meaning mentioned in pylint [documentation](http://pylint.pycqa.org/en/latest/how_tos/custom_checkers.html#write-a-checker). The other messages in this class are a bad example, look at the most recent checkers to get an idea.